LuaMenu/widgets/snd_music_lite.lua

```diff
@@ -154,13 +154,13 @@ function widget:Initialize()
 		return
 	end
 
-	OPEN_TRACK_NAME = WG.Chobby.Configuration.gameConfig.openTrack
-	if OPEN_TRACK_NAME ~= nil then
-		if OPEN_TRACK_NAME == nil or OPEN_TRACK_NAME == '' then
-			OPEN_TRACK_NAME = randomTrackList[math.random(#randomTrackList)]
+	openTrack = WG.Chobby.Configuration.gameConfig.openTrack
+	if openTrack ~= nil then
```

Comment:
I don't know what this nested if is meant to do, but there has to be a mistake here.

Reply:
Why is open track being checked against ''? It looks like the meaning of configuration values is being overridden in a confusing way.
- openTrack should be a string. If it is set then that track is played when the menu opens.
- If openTrack is not set then a random track should be played.
- If someone wants the ability to start the menu with no music they can add a config value to do that.
- If the configuration has openTrack = '' then the music widget should print a warning about that track not existing.